src/api.ts

```diff
@@ -1,11 +1,10 @@
 import { Todo } from './types/Todo';
 import { User } from './types/User';
 
-// eslint-disable-next-line max-len
-const BASE_URL = 'https://mate-academy.github.io/react_dynamic-list-of-todos/api';
+// eslint-disable-next-line operator-linebreak
```

Comment:
The ESLint rule `operator-linebreak` is disabled here. Consider refactoring the code to avoid disabling the rule, such as by using template literals or adjusting the line break.